etc/rose-bash-completion

```diff
@@ -57,7 +57,7 @@ _rose() {
     else
         ROSE_DIR=$(__rose_get_rose_dir)
     fi
-    ok_subcommands=$(cd $ROSE_DIR/bin && ls $rose_command-* | \
+    ok_subcommands=$(cd $ROSE_DIR/ && ls $rose_command-* | \
```

Comment:
This feels fishy.

Reply:
I think I'll have to talk to you about this.

Reply:
OK, diddn't realise that was what's going on, this is the same approach as used by Cylc.